log/logrus.go

```diff
@@ -63,6 +63,14 @@ func LogrusWithWriter(writer io.Writer) Option {
 	}
 }
 
+// Custom formatter can be used to change default formatting
```

Comment:
@scortier Ideally comment should start with `LogrusWithFormatter`

Reply:
Yes ,it will be more relevant 
Resolved